src/main/scala/chisel3/ltl/LTL.scala

```diff
@@ -159,6 +226,25 @@ object Sequence {
     lhs
   }
 
+  /** Form the conjunction of two sequences, where the start and end
+    * times of both sequences must be identical. Equivalent to
+    * `arg0 intersect arg1 intersect ... intersect argN` in SVA.
+    */
+  def intersect(arg0: Sequence, argN: Sequence*)(implicit sourceInfo: SourceInfo): Sequence = {
+    var lhs = arg0
+    for (rhs <- argN) {
+      lhs = OpaqueSequence(LTLIntersectIntrinsic(lhs.inner, rhs.inner))
+    }
+    lhs
+  }
```

Comment:
```suggestion
  def intersect(args: Sequence*)(implicit sourceInfo: SourceInfo): Sequence = {
    require(args.nonEmpty, s"intersect requires at least one argument! Got $args")
    args.tail.foldLeft(args.head)((lhs, rhs) =>
      OpaqueSequence(LTLIntersectIntrinsic(lhs.inner, rhs.inner))
    )
  }
```
It's more canonical to write this for loop with a fold.

I get that you're sort of forcing the user to have at least one argument via the arg followed by varargs, but that has the nasty side effect of making it very annoying to call this with a `Seq[Sequence]` argument, so I think it's best just to go full varargs.

Another alternative is to overload, have this form and then have a 2nd that takes a `Seq`.

Reply:
oh ok, yeah that makes sense, I was just following the pattern used for `and` and `or`. Should I update those too?

Reply:
In that case I think just leave it consistent (i.e. leave it as is), we can consider changing them all in a separate PR.